assets/pages/new-contributions-landing/contributionsLandingReducer.js

```diff
@@ -99,7 +102,7 @@ function createFormReducer(countryGroupId: CountryGroupId) {
   // ----- Initial state ----- //
 
   const initialState: FormState = {
-    contributionType: 'MONTHLY',
```

Comment:
You need to change this back to `MONTHLY`

Reply:
oops!